### DISCUSSION TITLE
Any workarounds to achieve on-demand revalidation?

### DISCUSSION THREAD

**Discussion by leandwo** [vercel/next.js#24017] [Help]
My team needs on-demand revalidation to complete our transition to next.js. The feature is on the roadmap but there is no RFC or timeline yet.

In the meantime, has anyone found a way to regenerate a single page path on request?

**Comment by michrome** [comment]
A low-tech (but could be ‘good enough’) approach might be to set a low `revalidate` number i.e. `1` then make an HTTP request to the page when you want it regenerated.

**Reply by sanzhardanybayev** [reply]
@michrome but that's not resource efficient. Next doesn't skip rebuild when gets the same data.

**Comment by sanzhardanybayev** [comment]
@leosolid when can I see the roadmap?

**Reply by michrome** [reply]
Hey @sanzhardanybayev I'd keep an eye on https://github.com/vercel/next.js/discussions/11552#discussioncomment-646963. I wouldn't be surprised if it's announced at [Next.js Conf](https://nextjs.org/conf#room-8fk2f) later this month.

Until then, use a low `revalidate` number, with the caveat that it's not resource efficient as per your comment above.
